src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectVotingController.kt

```diff
@@ -0,0 +1,99 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse403
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.VoteOption
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/projects/{projectId}/voting")
+@RestController
+class ProjectVotingController() {
+  @ApiResponse200
+  @ApiResponse403
+  @ApiResponse404
+  @GetMapping("/")
```

Comment:
It might be better to do `@GetMapping` (without the path) here so that `GET /api/v1/accelerator/projects/1234/voting` works. I believe with the slash here, you'd need to do `GET /api/v1/accelerator/projects/1234/voting/`.